tempesta_fw/http.c

```diff
@@ -446,9 +446,16 @@ tfw_http_conn_release(TfwConnection *conn)
  * Release resources that are not needed anymore, and keep other
  * resources that are needed while there are users of the connection.
  */
+static void tfw_http_resp_terminate(TfwHttpMsg *hm);
```

Comment:
It seems you declared the function here just to avoid `tfw_http_conn_drop()` movement. However, it seems it can be placed below `tfw_http_resp_terminate()`. Please use the declarations only when you absolutely need cyclic dependency (this is the first thing which comes to mind when I see predeclarations).